manifests/Arion-Air Superiority.yaml

```diff
@@ -0,0 +1,21 @@
+name: Arion - Air Superiority
+authors: Spectre907 (With help from 1010todd, yuyuko_saigyouji. and gefulltetaubenbrust)
```

Comment:
```suggestion
authors: Spectre907 (With help from 1010todd, yuyuko_saigyouji, and gefulltetaubenbrust)
```

Reply:
That one is on purpose, that is the persons discord username